From 46dca79c722c561df69aa87b87d18bd0c48d4de7 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 27 Nov 2015 15:04:06 +0800
Subject: [PATCH 0685/5242] MLK-11914-1 video: mipi_dsi: correct wrong buf
 index usage

commit  1922eabc8a7669947a6f61680ea18d7e0d1e5491 from
https://source.codeaurora.org/external/imx/linux-imx.git

The 'buf' index loop is obviously wrong. So correct
it.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c b/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
index 4de1967..8c0cf4f 100644
--- a/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
+++ b/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
@@ -284,7 +284,7 @@ int mipid_hx8369_lcd_setup(struct mipi_dsi_info *mipi_dsi)
 	buf[3] = HX8369_CMD_SETGAMMA_PARAM_4;
 	buf[4] = HX8369_CMD_SETGAMMA_PARAM_5;
 	buf[5] = HX8369_CMD_SETGAMMA_PARAM_6;
-	buf[7] = HX8369_CMD_SETGAMMA_PARAM_7;
+	buf[6] = HX8369_CMD_SETGAMMA_PARAM_7;
 	buf[7] = HX8369_CMD_SETGAMMA_PARAM_8;
 	buf[8] = HX8369_CMD_SETGAMMA_PARAM_9;
 	err = mipi_dsi_pkt_write(mipi_dsi, MIPI_DSI_GENERIC_LONG_WRITE, buf,
-- 
1.7.9.5

